### PR TITLE
Fix swapConfig persistency on Swap tab

### PR DESCRIPTION
**Pull request by PonyJackal**
WHAT'S DONE
- move init/update swapConfig into global.js
- load token pair from GlobalStateManager in global js

TODO
- need to fix swapConfig persistency on Bridge tab

### CODE REVIEW

**Comment by chillvybz** [comment]
This is a good start but I would like to see more cleanup of the /TODOs.
And also the bridge work.l

**Comment by yuripramos** [comment]
Please use the labels **WIP** or **ready to view** to put some context to other developers of your work.
Is this ready to review @PonyJackal  ?

**Comment by PonyJackal** [comment]
> Please use the labels **WIP** or **ready to view** to put some context to other developers of your work. Is this ready to review @PonyJackal ?

Yeah, it is ready to review,
Not sure why I was not able to edit this PR, can't add labels

File: src/client/js/components/partials/bridge/BridgeWidget.jsx [hunk 1]
```diff
@@ -41,21 +41,22 @@ export default class BridgeWidget extends Component {
...
     );
-    const bridgeConfig = GlobalStateManager.getBridgeConfig();
     
     const updatedConfig = {
       fromChain: network,
-      toChain: bridgeConfig?.toChain || toChain,
-      to: TokenListManager.findTokenById(bridgeConfig.to.symbol),
-      from: TokenListManager.findTokenById(bridgeConfig.from.symbol)
+      toChain,
+      to: TokenListManager.findTokenById(bridgeConfig.to.symbol).address ?  TokenListManager.findTokenById(bridgeConfig.to.symbol) :  TokenListManager.findTokenById(
```
Comment (chillvybz) [inline src/client/js/components/partials/bridge/BridgeWidget.jsx:52]: `TokenListManager.findTokenById(...)` may return undefined.
So please protect `TokenListManager.findTokenById(...).address` against undefined.

File: src/client/js/components/partials/bridge/BridgeWidget.jsx [hunk 2]
```diff
@@ -41,21 +41,22 @@ export default class BridgeWidget extends Component {
...
+        toChain.supportedCrossChainTokens[0],
+        toChain,
+      ),
+      from: TokenListManager.findTokenById(bridgeConfig.from.symbol).address ? TokenListManager.findTokenById(bridgeConfig.from.symbol) : TokenListManager.findTokenById(network.supportedCrossChainTokens[0])
```
Comment (chillvybz) [inline src/client/js/components/partials/bridge/BridgeWidget.jsx:56]: Same comment here about undefined

**Pull request merged**